docs/docs/cmd/login.md

```diff
@@ -25,9 +25,6 @@ m365 login [options]
 `-c, --certificateFile [certificateFile]`
 : Path to the file with certificate private key. Required when `authType` is set to `certificate`
 
-`--thumbprint [thumbprint]`
```

Comment:
Let's not remove this option as it would break backwards compatibility. We need to leave the current functionality as-is in v3. We could deprecate it and mark for removal in v4, but we can't just do it right now.

Since the thumbprint is optional, we can keep using it. If it's provided, we don't need to discover it ourselves. If it's not provided, we'll get it ourselves.

Reply:
Ok, I'll put it back as allow options but I still remove it from the guidance documentation.